src/Search/Annotation/GearmanTaskDriver.php

```diff
@@ -4,8 +4,8 @@
 
 use Closure;
 use Doctrine\Common\Annotations\Reader;
+use eLife\Logging\Monitoring;
 use eLife\Search\Gearman\InvalidWorkflow;
-use eLife\Search\Monitoring;
```

Comment:
if you change all these references you can now remove the class `eLife\Search\Monitoring`